R/CohortConstruction.R

```diff
@@ -627,4 +629,25 @@ saveAndDropTempInclusionStatsTables <- function(connection,
                                                progressBar = FALSE,
                                                reportOverallTime = FALSE,
                                                oracleTempSchema = oracleTempSchema)
+}
+
+
+.warnMismatchSqlInclusionStats <- function(sql) {
+  if (any(stringr::str_detect(string = sql, pattern = "_inclusion_result"), 
+          stringr::str_detect(string = sql, pattern = "_inclusion_stats"), 
+          stringr::str_detect(string = sql, pattern = "_summary_stats")
+          )
+  ) {
+    if (isFALSE(generateInclusionStats)) {
```

Comment:
Are you referring to a global variable (`generateInclusionStats`) here?!?

This should be an argument of the `.warnMismatchSqlInclusionStats` function

Reply:
Thank you !

Reply:
Thank you! 

```
.warnMismatchSqlInclusionStats: no visible binding for global variable
  â€˜generateInclusionStatsâ€™
launchCohortExplorer: no visible binding for global variable
  â€˜shinySettingsâ€™
Undefined global functions or variables:
  generateInclusionStats shinySettings
```

```
* checking dependencies in R code ... WARNING
'::' or ':::' import not declared from: â€˜stringrâ€™
```